terraform/ecs/provision/restarts.tf

```diff
@@ -96,7 +96,7 @@ resource "local_file" "app_no_slack" {
 
 resource "null_resource" "package_slack_sdk" {
   provisioner "local-exec" {
-    interpreter = ["/bin/bash", "-c"]
+    interpreter = ["/bin/sh", "-c"]
```

Comment:
Why not add `bash` to the dockerfile because that's what's on our production environment?